_data/beyond-sidebar.yml

```diff
@@ -1064,6 +1064,14 @@
             - title: List cross-selling products
               link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-list-cross-sell-products
               id: list_cross_selling_products
+        - title: Product attachments
+          entries:
+            - title: List product attachments
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-attachments-list
+              id: list_attachments
+            - title: Show product attachment details
```

Comment:
Related comment: https://github.com/ePages-de/ng-product-view/pull/689/files#r1048490891

Reply:
see related comment in the same PR 😁